spec/requests/v1/tasks_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
```

Comment:
Same as above: 
1. replace all http statuses from number to symbol
2. use ffaker instead of hardcoded values